brokers/vscode/broker.go

```diff
@@ -135,9 +129,31 @@ func (b *Broker) processPlugin(meta model.PluginMeta) error {
 		}
 	}
 
+	image := meta.Attributes["containerImage"]
```

Comment:
Should not you check whether attributes are not nil as you did it before extension evaluation https://github.com/eclipse/che-plugin-broker/pull/31/files#diff-f9c03892980ba78514d5ef65ae08159eR100 ?

Reply:
It is not needed anymore. If attributes are not present but URL is present we consider plugin local and hosted not in VS Code marketplace